### PR TITLE
Add unit tests for FL install default rules

### PR DESCRIPTION
**Pull request by Levitan**
Issue #749

---
This change is [Reviewable](https://reviewable.io/reviews/telstra/open-kilda/1481)

### CODE REVIEW

**Comment by nikitamarchenko** [comment]
LF?

File: services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml [hunk 1]
```diff
@@ -232,5 +232,13 @@
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="109" checks="JavadocMethod"/>
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="118" checks="AbbreviationAsWordInName"/>
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="118" checks="JavadocMethod"/>
+    <suppress files="src/test/java/org/openkilda/floodlight/test/standard/OutputCommands.java" lines="83" checks="JavadocMethod"/>
```
Comment (nikitacherevko) [inline services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml]: Would be better to fix checkstyle issues, not just ignore them

Comment (nikitacherevko) [reply services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml]: Seems it is test class, so I guess there is no need to write docs

File: services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml [hunk 2]
```diff
@@ -173,4 +173,12 @@
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="109" checks="JavadocMethod"/>
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="118" checks="AbbreviationAsWordInName"/>
     <suppress files="src/test/java/org/openkilda/floodlight/FloodlightTestCase.java" lines="118" checks="JavadocMethod"/>
+    <suppress files="src/test/java/org/openkilda/floodlight/test/standard/OutputCommands.java" lines="85" checks="JavadocMethod"/>
```
Comment (sergii-iakovenko) [inline services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml]: It’s agreed that we don’t add suppressions anymore instead we fix the issues.

File: services/src/floodlight-modules/src/test/java/org/openkilda/floodlight/switchmanager/SwitchManagerTest.java [hunk 3]
```diff
@@ -115,6 +115,7 @@
     private SwitchDescription switchDescription;
     private DatapathId dpid;
     private SwitchManagerConfig config = new Config();
+    private DatapathId defaultDpid;
```
Comment (sergii-iakovenko) [inline services/src/floodlight-modules/src/test/java/org/openkilda/floodlight/switchmanager/SwitchManagerTest.java]: Shouldn’t it be a constant?

Comment (Levitan) [reply services/src/floodlight-modules/src/test/java/org/openkilda/floodlight/switchmanager/SwitchManagerTest.java]: Fixed

File: services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml (hunk 2 above)
Comment (Levitan) [reply services/src/floodlight-modules/src/checkstyle/checkstyle-suppress-known-issues.xml]: Add javaDocs for my method, and remove supress

**Comment by Levitan** [comment]
> refactor duplicate code, please

refactored

**Pull request merged**